steps/steps.go

```diff
@@ -37,7 +37,6 @@ func stepListItem(stepIDComposite, title, runIf string, inputs ...envmanModels.E
 	}
 }
 
-// DefaultPrepareStepList ...
```

Comment:
The removal of these comments indicates that we are not running the same CI stack as we do for our steps. I would probably keep it for now and instead hook this up to the regular workflow. (But obviously not in this PR.)

Reply:
We actually do! I made this PR a few days ago: https://github.com/bitrise-io/bitrise-init/pull/248

I wanted to get rid of these dummy comments, which are enforced by the old `golint` project. That is replaced with `golangci-lint`, the way we lint step repos.